pkg/private/tar/tar.bzl

```diff
@@ -272,6 +274,10 @@ pkg_tar_impl = rule(
         ),
         "create_parents": attr.bool(default = True),
         "allow_duplicates_from_deps": attr.bool(default = False),
+        "compress_level": attr.string(
+            doc = """Specify the numeric compress level in gzip mode; may be 0-9 or empty (6).""",
```

Comment:
Why as string rather than an int? The docstring is given in terms of an int

Reply:
I made it string, so that its default value can be not `0` but an empty string. Do I need to make the attr to be `6` by default?

Reply:
People usually make the default -1 for ints and then figure out what do to.  So you can make the behavior work well either way.

The real question is what is easier for the user. I can make a case for either. If this were low, medium, high, then a string for sure.  zip uses 0-9, so an int seems right. Another compression type might use different level meanings.

If you had wanted to use an int, but only did it for the default, then you can make it that. But I suspect this won't matter because most people will use the default. We can even change it in the macro wrapper so that people could use either. :-)

I'll merge it either way. Just LMK

Reply:
Hello, today I noticed the name `compression_level` in ZIP-related code, so I updated this PR and now the attribute is `compression_level: int = -1`.